_includes/footer.html

```diff
@@ -1,7 +1,7 @@
 {% include base_path %}
 
 <div class="page__footer-follow text-center">
-  <p style="margin-bottom: 0.5em"><a href="{{ site.baseurl }}/community/">Made with :heart: by :octocat:s and friends</a></p>
+  <p style="margin-bottom: 0.5em"><span aria-label="Made with love by GitHub and friends"><a href="{{ site.baseurl }}/community/">Made with love :heart: by GitHub :octocat: and friends</a></span></p>
```

Comment:
@brianamarie using the Wave Chrome extension, I see that screenreaders would still read both the included `alt` text and the `aria-label`. 

<img width="955" alt="screen shot 2017-08-24 at 8 50 55 am" src="https://user-images.githubusercontent.com/16547949/29667159-5f3c7f74-88a9-11e7-9af5-43881b5bba58.png">

I think we'd have greater positive impact if we bring this issue up on [jekyll/jemoji](https://github.com/jekyll/jemoji) directly and see what they say. If we can introduce a fix there, it'd solve the problem for anyone using that Gem, not just us.

Reply:
@hectorsector I think that's a great idea. 

What are your thoughts on merging this PR as is anyway as a temporary fix? I am happy to take responsibility for adding this idea to the backlog, but I'm concerned that if I take responsibility for opening a fix for the gem, it would fall through the woodworks due to bandwidth.

Reply:
👍  issue opened in Jemoji.